### ISSUE TITLE
Improve sequelize-sscce

### ISSUE THREAD

**Issue by ephys** [sequelize/sequelize#13913]
Current SSCCE template has a couple of issues that we should resolve to make testing & creating SSCCE smoother:

- Fix `log` crashing when encountering a `bigint`.
- Add docker containers for each dialect for local testing.
- Add a script to test for each dialect, for local testing.
- Remove the .js files and only include the .ts files, to reduce confusion as to which one you should be editing.

**Comment by WikiRik** [comment]
@papb Thanks again for transferring the SSCCE repo to the organization!

**Comment by WikiRik** [comment]
I just added run the Actions for the first time and the TS checks are broken, something in the `ts-prep` script. So that also needs to be looked into

**Comment by papb** [comment]
Cool! By the way, when I first created papb/sequelize-sscce, we didn't have the [`dev` folder](https://github.com/sequelize/sequelize/tree/main/dev). Months later, I finally decided to rework executing things locally from the main repo itself, and created the `dev` folder. By then, it crossed my mind that perhaps papb/sequelize-sscce would not even be really necessary anymore. Although it is still good for someone without Docker, maybe in their phone or something?
